docker/compose-spec-l2-services.yml

```diff
@@ -126,39 +126,6 @@ services:
       linea:
         ipv4_address: 11.11.11.119
 
-  traces-node:
-    container_name: traces-node
-    hostname: traces-node
-    image: consensys/linea-geth:${ZKGETH_TAG:-0588665}
-    platform: linux/amd64
-    profiles: [ "l2", "l2-bc", "debug", "external-to-monorepo" ]
-    depends_on:
-      sequencer:
-        condition: service_healthy
-    ports:
-      - "8645:8545" # http
-      - "8646:8546" # websockets
-      - "8651:8551" # engine_ API
-    environment:
-      DATA_DIR: "/l2-zkgeth-traces-node/"
-      NETRESTRICT: "11.11.11.0/24"
-      # ETHSTATS_URL: "zkgeth-traces-node:${WS_SECRET:-dummy-secret}@l2-ethstats:3000"
-      BOOTNODES: "enode://14408801a444dafc44afbccce2eb755f902aed3b5743fed787b3c790e021fef28b8c827ed896aa4e8fb46e22bd67c39f994a73768b4b382f8597b0d44370e15d@11.11.11.101:30303"
-      TRACES_DIR: "/data/traces/raw"
-      MAX_BLOCK_GAS: 10000000  #10M
-      MAX_BLOCKDATA_BYTES: 35000 #35 KBytes
-      MAX_TXDATA_BYTES: 30000 #30 KBytes
-      VERSION: "0.2.0"
-      TRACES_FILE_EXTENSION: ".json.gz"
-    entrypoint: [ "sh", "/scripts/node.sh", "1337", "/genesis.json", "0xa", "0x1C9C380", "0xF4240" ]
-    volumes:
-      - ./geth/scripts:/scripts:ro
-      - ./config/linea-local-dev-genesis-PoA-geth.json:/genesis.json:ro
-      - ../tmp/local/:/data/
-    networks:
-      linea:
-        ipv4_address: 11.11.11.102
-
   traces-node-v2:
```

Comment:
I think we should just rename it to `traces-node`

Reply:
Make sense, will add that in next PR